### PR TITLE
Adding news section and new logos

### PR DESCRIPTION
**Pull request by nate-double-u**
Adding a News section, and adding logos for 56k Cloud, AWS IoT Greengrass, NetFoundry, and Scalys, as per https://cncfservicedesk.atlassian.net/browse/CNCFSD-1030

### CODE REVIEW

**Comment by nate-double-u** [comment]
Hold for updated logo.

**Comment by nate-double-u** [comment]
Unhold, logo updated.

**Comment by nate-double-u** [comment]
I think this is ready for review.
/cc @chalin @caniszczyk

File: content/_index.md [hunk 1]
```diff
@@ -45,6 +45,20 @@ title: "Parsec"
   </div>
 {{< /jumbotron >}}
 
+{{< jumbotron title="News / Announcements" bgcolor="#e9ecef" textcolor="#000000" align="text-left" >}}
```
Comment (chalin) [inline content/_index.md]: Maybe just use the title "News"?

File: content/_index.md [hunk 2]
```diff
@@ -45,6 +45,20 @@ title: "Parsec"
...
+
+<div class="mt-5 row features">
+  <div class="col col-md-12 col-12">
+    <ul>
+    <li><a href="https://netfoundry.io/secure-networking-for-connected-and-autonomous-vehicles/">Secure networking for connected and autonomous vehicles</a> &mdash; November 30, 2021</li>
+    <li><a href="https://community.arm.com/arm-community-blogs/b/tools-software-ides-blog/posts/parsec-adds-security-for-aws-iot-greengrass">Parsec adds a new level of security for AWS IoT Greengrass</a> &mdash; November 23, 2021</li>
+    <li><a href="https://azure.microsoft.com/en-us/blog/the-enclave-device-blueprint-for-confidential-computing-at-the-edge/">The Enclave Device Blueprint for confidential computing at the edge</a> &mdash; October 19, 2021</li>
+    </ul>
```
Comment (chalin) [inline content/_index.md:56]: This is a markdown file so you could define the list using markdown (provided you change the shortcode processing markers from `<` and `>` to `%`). Markdown would be more readable and easier for folks to maintain.

Oh, you'll need to unindent the HTML too.

File: layouts/shortcodes/logos.html [hunk 3]
```diff
@@ -2,36 +2,48 @@
   <div class="mt-4 col col-md-12 col-12">
     <h2 class="mb-3">Parsec is an open source collaborative project</h2>
   </div>
-  <div class="mt-4 col col-md-3 col-6">
+  <div class="mt-4 col col-md-3 col-6" style="text-align:center">
```
Comment (chalin) [inline layouts/shortcodes/logos.html:5]: Consider defining the `text-align:center` styling using SASS rather than hard coding the styling on each element.

**Review by chalin — APPROVED** [review]
Looks fine. See inline comments for suggested improvements, which can me made via another PR or not at all (if you feel that it's just fine like that).

File: layouts/shortcodes/logos.html (hunk 3 above)
Comment (nate-double-u) [reply layouts/shortcodes/logos.html:5]: I think that might actually be what [line 1](https://github.com/parallaxsecond/parsec-website/blob/4c7f59c70fbda0ab8c14bff0042afeb535ba4246/layouts/shortcodes/logos.html#L1) may be trying to do with `align-items-center`.

**Pull request merged**